src/web/app/pages-logs/logs-page.component.ts

```diff
@@ -248,12 +308,19 @@ export class LogsPageComponent implements OnInit {
       .replace(/\t/g, '&#9;');
   }
 
+  /**
+   * Remove the resource name infront.
+   */
+  private formatTraceForSummary(trace: string): string | undefined {
+    return trace.split('/').pop();
```

Comment:
This is good but can be shortened even further. Looking at the first 7-9 digits is good enough to at a glance to group the logs from same HTTP request together, much like how git commits are usually shortened to 7-9 digits when displayed.

Reply:
Will update this part to display only the first 9 digits